sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -182,10 +182,13 @@ class SQLMetricsSuite extends SparkFunSuite with SQLMetricsTestUtils with Shared
   }
 
   test("Sort metrics") {
-    // Assume the execution plan is
-    // WholeStageCodegen(nodeId = 0, Range(nodeId = 2) -> Sort(nodeId = 1))
-    val ds = spark.range(10).sort('id)
-    testSparkPlanMetrics(ds.toDF(), 2, Map.empty)
+    // Assume the execution plan with node id is
+    // Sort(nodeId = 0)
+    //   Exchange(nodeId = 1)
+    //     LocalTableScan(nodeId = 2)
+    val df = Seq(1, 3, 2).toDF("id").sort('id)
+    testSparkPlanMetrics(df, 2, Map.empty)
```

Comment:
Thanks for pinging me @maropu. What is the point about checking that `LocalTableScan` contains no metrics?

I checked the original PR which introduced this UT by @sameeragarwal who can maybe help us stating the goal of the test here (unless someone else can answer me, because I have not understood it). It doesn't seem even related to the Sort operator to me. Maybe I am missing something.

Reply:
@mgaido91 This case tries to check `Sort` (nodeId=0) metrics, rather than `LocalTableScan`. The second parameter (`2`) of `testSparkPlanMetrics(df, 2, Map.empty)` means `expectedNumOfJobs` rather than `nodeId`. The third parameter `expectedMetrics` will pass `nodeId` together with corresponding expected metrics. Because metrics of Sort node (including `sortTime`, `peakMemory`, `spillSize`) may change during each execution, unlike metrics like `numOutputRows`, we have no way to check these values.

Reply:
can we check the metrics of `SortExec` here?

Reply:
@cloud-fan This case tries to check metrics of `SortExec`, however these metrics (`sortTime`, `peakMemory`, `spillSize`) change each time  the query is executed, they are not fixed. So far what I did is to check whether `SortExec` exists. Do you mean we should further check whether these metrics names exist? Though we can't know their values beforehand.

Reply:
can we just check something like `sortTime  > 0`?

Reply:
+1 for @cloud-fan suggestion. I mean, if we cannot check their exact value, we should at least check that they exist/have reasonable values. Otherwise this UT is useless.

Reply:
This makes sense! Let me try.

Reply:
@seancxmao what do you think about updating this?

Reply:
@srowen Thanks for reminding me. I'll update this later this week. So busy these days...